intermediate_source/dist_tuto.rst

```diff
@@ -63,8 +63,9 @@ the following template.
     if __name__ == "__main__":
         size = 2
         processes = []
+         mp.set_start_method("spawn")
```

Comment:
wrong indent?

Reply:
uff, sorry, my bad. Corrected. Waiting for tests